public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
+  void log(dynamic msg) {
+    window.console.log(msg);
+  }
+
+  void error(dynamic msg) {
```

Comment:
Yeah that was just to be consistent between the methods. I thought using `print` for the first one then `window.console.*` would seem a bit confusing. To be honest though Im not sure what the point of the logger service is even for. I don't see a reference to it anywhere. I wonder if it's in the works for one of the upcoming examples